security-team/security-groups.md

```diff
@@ -0,0 +1,37 @@
+## KubeEdge security mailing lists
+
+File an issue [here](https://github.com/kubeedge/community/issues/new?template=distributors-application.md), filling in the criteria template to join [cncf-kubeedge-distrib-announce@lists.cncf.io](mailto:cncf-kubeedge-distrib-announce@lists.cncf.io).
+
+### Distributors announce
+
+Email:
+
+cncf-kubeedge-distrib-announce@lists.cncf.io
+
+Owners:
+
+- [wangzefeng@huawei.com](mailto:wangzefeng@huawei.com)
+
+- [xufei40@huawei.com](mailto:xufei40@huawei.com)
+- [linguohui1@huawei.com](mailto:linguohui1@huawei.com)
+
+Members:
+
+*TODO*
+
+### The Security Team
+
+Email:
+
+cncf-kubeedge-security@lists.cncf.io
+
+Owners:
+
+- [wangzefeng@huawei.com](mailto:wangzefeng@huawei.com)
+
+- [xufei40@huawei.com](mailto:xufei40@huawei.com)
+- [linguohui1@huawei.com](mailto:linguohui1@huawei.com)
+
+Members:
+
+*TODO*
```

Comment:
As a starting point, we can just list initial members here which are actually intial owners